_docs/project-ideas.md

```diff
@@ -0,0 +1,25 @@
+Stuck for ideas? You're not alone. Even the creative geniuses of our time need places to start to get some inspiration. Here are some ideas to help you get started.
```

Comment:
Both pages will need the jekyll headers so they populate in the menu. See [the template file](https://raw.githubusercontent.com/UWB-ACM/Hackathon-Docs-2020/master/_docs/template.md) for the raw syntax.

```suggestion
---
layout: default
category: Getting Started
title: Project Ideas
order: 4
permalink: /project_ideas.html
---

# Project Ideas

Stuck for ideas? You're not alone. Even the creative geniuses of our time need places to start to get some inspiration. Here are some ideas to help you get started.
```